package.json

```diff
@@ -22,7 +22,7 @@
     "lint:fix": "eslint . --cache --max-warnings 0 --fix && node_modules/.bin/shellcheck ./script/**/*[^utils] ./script/utils/**",
     "typecheck": "tsc && tsc -p integration_tests",
     "test": "jest --collectCoverage",
-    "test:ci": "jest --collectCoverage --maxWorkers=50%",
+    "test:ci": "jest --collectCoverage --runInBand --coverageThreshold=\"{}\"",
```

Comment:
What does this coverage threshold mean?

Reply:
This allows us to create the coverage report, but not check the coverage, as we're only running a small subset of tests in each container. They then get persisted to the workspace as `./coverage/coverage_${CIRCLE_NODE_INDEX}.json` and joined together with `nyc merge`

Reply:
makes sense